zerver/lib/actions.py

```diff
@@ -822,6 +822,10 @@ def do_send_messages(messages_maybe_none):
                 if user_profile.id in mentioned_ids:
```

Comment:
I just realized one more thing.  Can you please fix the comment above that says "Currently, we only support mentions as triggers"?